### PR TITLE
AGSEC-206/AGSEC-207 - Update the security landing page for ag.org 2.0

### PR DESCRIPTION
**Pull request by abstractj**
Good morning, this PR update our security landing page for something more realistic. I'm still working at the guides, but decided to submit it for immediate feedback.

Updates:
- Removing html and replacing by markdown
- Rephrasing some sentences at the main page
- Organizing sections

@danielpassos @cvasilak @lfryc would you mind to review?

### CODE REVIEW

**Comment by abstractj** [comment]
I update this pull request to include the sections which I'm working for the guide. Also "In progress" was included and will be removed before we ship it. I'm still working on the other sections.

**Comment by danielpassos** [comment]
The `Get started` button seems a little bigger, not sure if it's correct

**Comment by danielpassos** [comment]
Not sure if it's related of this PR but the roadmap needs some design improvements

**Comment by abstractj** [comment]
@danielpassos the roadmap is based on Jira. Not sure exactly what do you mean, could you please be more specific?

**Comment by danielpassos** [comment]
The bigger button

**Comment by danielpassos** [comment]
About the roadmap, the page layout/design seems strange for me

screenshot 2015-03-06 07 20 22

**Comment by abstractj** [comment]
@danielpassos regarging the bigger button, this is a patter http://staging-aerogearsite.rhcloud.com/push/.

Regarding the screenshot, that's not the security roadmap and also not part of this change. @matzew created that page a it will be moved.

**Comment by matzew** [comment]
> @matzew created that page a it will be moved.

Not sure I understand

**Comment by abstractj** [comment]
@matzew OAuth2 page will be moved to the proper section, that's all :)

**Comment by matzew** [comment]
ah! :-) cool!

On Fri, Mar 6, 2015 at 12:10 PM, Bruno Oliveira <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> OAuth2 page will be moved to the
> proper section, that's all :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/532#issuecomment-77543252>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by lfryc** [comment]
@danielpassos the `Getting Started` is correct in terms of consistency with other pages

Nice start, @abstractj ! Not sure which is still in progress, but I'll try:
* let's rename Roadmap button in header OR rather move it to appropriate subsection of the doc (under OAuth2 links?) - it's strange when general button (Roadmap) links to specific section (OAuth2)
* `Guides` button is incorrect as already reported by @danielpassos

File: getstarted/guides/index.markdown [hunk 1]
```diff
@@ -47,20 +47,23 @@ Designed to get you started with enterprise mobile development.
...
-<h4 id="securityAndroid">Android</h4>
-* [Authentication](/docs/guides/aerogear-android/auth)
-* [Authorization](/docs/guides/aerogear-android/authz)
+In progress
 
-<h4 id="securityIOS2">iOS</h4>
-* [Authorization](/docs/guides/aerogear-ios-2.X/Authorization)
+#### 2. OTP
+* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
```
Comment (cvasilak) [inline getstarted/guides/index.markdown]: small link typos:
s/ (AeroGear-OTP) / (/docs/guides/AeroGear-OTP)
s/ (aerogear-cordova/AerogearCordovaOTP) / (/docs/guides/aerogear-cordova/AerogearCordovaOTP)

File: getstarted/guides/index.markdown [hunk 2]
```diff
@@ -47,20 +47,23 @@ Designed to get you started with enterprise mobile development.
...
 
-<div class="alert alert-info" role="alert">If you are looking for AeroGear iOS 1.x series, please visit <a href="/docs/guides/aerogear-ios/">this page</a> for more information.</div>
```
Comment (cvasilak) [inline getstarted/guides/index.markdown]: feel free to remove this '1.x series note' is already mentioned in up in the page and doesn't add any value

**Comment by cvasilak** [comment]
I really like the consolidation of the 'OAuth2 setup for various providers" in the "Before you get started" page, really nice

**Comment by abstractj** [comment]
@lfryc updated and rebased after your considerations. I missed the damn buttons mentioned by @danielpassos.

@cvasilak thanks bro, I appreciate it. Updated the PR with your considerations

**Comment by lfryc** [comment]
:+1: for merging

@danielpassos @cvasilak can I merge this?

**Comment by abstractj** [comment]
Rebased after the latest updates.

**Comment by danielpassos** [comment]
@lfryc :+1:

**Comment by lfryc** [comment]
Merging...

**Comment by lfryc** [comment]
great job @abstractj 

landed :rocket:  on the :moon:

**Pull request merged**